modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -131,6 +131,12 @@ public void onAfterUpdate(
 			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
 			originalObjectEntry, objectEntry);
 
+		_runRelevantObjectEntryModelListeners(
```

Comment:
Why do we need this change?

Reply:
Because if the ERC of the updated object entry is the same as original one, it will execute the `return` statement and it won't execute our API ModelListeners